client/checkout/classic/upe-deferred-intent-creation/payment-processing.js

```diff
@@ -206,16 +276,23 @@ export const checkout = ( api, jQueryForm, paymentMethodType ) => {
 			await validateElements( elements );
 			const paymentMethodObject = await createStripePaymentMethod(
 				api,
-				elements
+				elements,
+				jQueryForm
 			);
 			appendFingerprintInputToForm( jQueryForm, fingerprint );
 			appendPaymentMethodIdToForm(
 				jQueryForm,
 				paymentMethodObject.paymentMethod.id
 			);
+			await additionalActionsHandler(
+				jQueryForm,
+				paymentMethodObject.paymentMethod,
+				api
+			);
```

Comment:
```suggestion
			await additionalActionsHandler(
				paymentMethodObject.paymentMethod,
				jQueryForm,
				api
			);
```
I suggested swapping these two parameters in another comment, since only the payment method is required by all callback functions that can apply here. Just spelling it out here as a complementary comment, just in case my previous explanation was confusing beyond clarity. 😅